features/visitor_is_able_to_sign_up.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Visitor is able to sign up
+    As a User
+    In order to access the mailboxer
+    I want to be able to Sign Up
+
+Scenario: User can sign up
+
+        Given I visit the page
```

Comment:
The first three steps of every scenario are the same, you should move them to a background

Reply:
done